src/Transition/CreateNewIDFUsingRules_template.f90

```diff
@@ -11,7 +11,13 @@ SUBROUTINE SetThisVersionVariables()
       ! TODO: Update this section as appropriate
       VerString='Conversion 9.3 => 9.4'
       VersionNum=9.4
-      sVersionNum='9.4'
+      ! Starting with version 22.1, the version string requires 4 characters
+      ! The original sVersionNum variable is a 3 character length string
+      ! If we just change that variable to be 4 characters, it could break everything before 22.1
+      ! So instead, let's just move forward with a new 4 character string and use that in this file and the future
+      ! If we get to version 100.1 and we are still using this Fortran transition then well....we can deal with it then
+      sVersionNum = '***'
+      sVersionNumFourChars='23.2'
```

Comment:
Updated the transition code template. @Myoldmopar this might break the version update script?

Reply:
No it should be fine I think.